cmd/consumer-audit/main.go

```
@@ -87,6 +91,34 @@ func main() {
+			ExtensionsRepo: *extensionsRepo,
+			MesheryRepoUI:  *mesheryRepoUI,
+			CloudRepoUI:    *cloudRepoUI,
+		})
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "consumer-audit: superseded report: %v\n", err)
+			os.Exit(1)
+		}
+		printSupersededReport(out, sup)
+
+		if *supersededEnforce && sup.HasSupersededUsage() {
+			fmt.Fprintln(out)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintln` is not checked

(errcheck)